Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs

```diff
@@ -990,15 +990,23 @@ public override bool CanConvert(Type objectType)
         }
 
         /// <summary>
-        /// Distributed Tracing Options. <see cref="Microsoft.Azure.Cosmos.DistributedTracingOptions"/>
-        /// </summary>
-        internal DistributedTracingOptions DistributedTracingOptions { get; set; }
-
+        /// Distributed Tracing Options. <see cref="Microsoft.Azure.Cosmos.DistributedTracingOptions"/> are only applicable when EnableDistributedTracing flag is true.
+        /// </summary>
+#if PREVIEW
+        public
+#else
+        internal
+#endif
+            DistributedTracingOptions DistributedTracingOptions { get; set; }
+        
         /// <summary>
-        /// Gets or sets value indicating whether distributed tracing activities (<see cref="System.Diagnostics.Activity"/>) are going to be created for the SDK methods calls and HTTP calls.
-        /// By default true for Preview package
+        /// Gets or sets value indicating whether distributed tracing activities (<see cref="System.Diagnostics.Activity"/>) are going to be created for the SDK operation calls and HTTP calls.
```

Comment:
```suggestion
        /// Gets or sets value indicating whether distributed tracing is enabled.
```

The extra information should be a `<remark>`.

```
<remark>
When enabled, activities (<see cref="System.Diagnostics.Activity"/>) are going to be created for the SDK operations. The Open Telemetry uses the "Azure.Cosmos" namespace.
</remark>
```

Or something similar. Remember this needs to be understood by a user.